packages/opentelemetry-core/src/common/time.ts

```diff
@@ -99,6 +99,15 @@ export function hrTimeDuration(
   return [seconds, nanos];
 }
 
+// Convert hrTime to timestamp.
```

Comment:
Nit: should this be JSDoc /** */ style comment?

Reply:
hmm all the file looks like this, so that's why i kept like this

Reply:
@draffensperger do you think all file should be changed then ?

Reply:
Yeah, I think ideally, though it's only a small stylistic thing